CHANGELOG.md

```diff
@@ -48,6 +48,7 @@ To learn more about active deprecations, we recommend checking [GitHub Discussio
 - **CPU/Memory scaler**: Add support for scale to zero if there are multiple triggers([#4269](https://github.com/kedacore/keda/issues/4269))
 - **Redis Scalers**: Allow scaling using redis stream length ([#4277](https://github.com/kedacore/keda/issues/4277))
 - **General:** Introduce new Solr Scaler ([#4234](https://github.com/kedacore/keda/issues/4234))
+- **Kafka Scaler:** Add support for OAuth extensions ([#4486](https://github.com/kedacore/keda/pull/4486))
```

Comment:
Could you please move this to Improvements section (sorted alphabetically) and create a new issue with the feature request and link the issue here instead of the PR?